README

```diff
@@ -16,12 +16,13 @@ plugins if desired.
 
 Enchant currently works with the following spell-checkers:
 
+    * Nuspell
```

Comment:
Please move this underneath Hunspell (I'd like to have the providers roughly in order of frequency of use).

Reply:
https://github.com/nuspell/enchant/commit/1739563777b88c5cb0203bb33ca77d784ca06d8b#diff-c47c7c7383225ab55ff591cb59c41e6bR20